install/0000_30_machine-api-operator_10_service.yaml

```
@@ -18,3 +18,32 @@ spec:
   selector:
     k8s-app: machine-api-operator
   sessionAffinity: None
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: machine-api-controllers
+  namespace: openshift-machine-api
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: machine-api-controllers-tls
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
+  labels:
+    k8s-app: controller
+spec:
+  type: ClusterIP
+  ports:
+  - name: machine-metrics
+    targetPort: machine-metrics
+    port: 8081
+  - name: controller-metrics
+    targetPort: controller-metrics
```

Should this be machineset rather than controller?
```suggestion
  - name: machineset-metrics
    targetPort: machineset-metrics
```